modules/apps/web-experience/layout/layout-taglib/src/main/resources/META-INF/resources/layouts_tree/js/layouts_tree_state.js

```diff
@@ -280,7 +280,7 @@ AUI.add(
 						node.get('children').forEach(A.bind(instance._restoreCheckedNode, instance));
 					},
 
-					_updateCheckedNodes: function(node, state) {
+					_updateCheckedNodes: function(node, state, force) {
```

Comment:
When does state can be undefined? Only when you set it as undefined at line 332 or somewhere else?